pkg/virt-controller/watch/workload-updater/workload-updater.go

```diff
@@ -352,6 +352,9 @@ func (c *WorkloadUpdateController) shouldAbortMigration(vmi *virtv1.VirtualMachi
 	if isVolumesUpdateInProgress(vmi) {
 		return false
 	}
+	if vmi.Status.MigrationState != nil && vmi.Status.MigrationState.TargetNodeDomainReadyTimestamp != nil {
```

Comment:
The only thing I wonder is when do we clear this and if that can affect this logic. Can we have a look at this?